src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -0,0 +1,128 @@
+// Dependencies
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { find, filter, get, startsWith, map, orderBy } from 'lodash';
+
+class SideNav extends Component {
+  static propTypes = {
+    navItemsLookup: PropTypes.object.isRequired,
```

Comment:
I would like to! But I can't predict what the keys will be on the top-level object since `navItemsLookup` is keyed off by ID, like:
```json
{
  "5c63e2e3-cd1b-4a30-8338-721b336f0639": {
    description: '',
    label: '',
    ...,
  },
}
```

Reply:
Ah I see! Makes sense 👍